worker/dbaccessor/worker.go

```diff
@@ -516,43 +533,64 @@ func (w *dbWorker) processAPIServerChange(apiDetails apiserver.Details) error {
 	ctx, cancel := w.scopedContext()
 	defer cancel()
 
-	// If this is an existing node, check if we are bound to the loopback IP
-	// and entering HA, which requires a new local-cloud bind address.
 	if extant {
 		asBootstrapped, err := mgr.IsBootstrappedNode(ctx)
 		if err != nil {
 			return errors.Trace(err)
 		}
 
-		// If this is a node that is already clustered, there is nothing to do.
-		if !asBootstrapped {
-			w.cfg.Logger.Debugf("node already clustered; no work to do")
-			return nil
+		serverCount := len(apiDetails.Servers)
+
+		// If we are as-bootstrapped, check if we are entering HA and need to
+		// change our binding from the loopback IP to a local-cloud address.
+		if asBootstrapped {
+			if serverCount == 1 {
+				// This bootstrapped node is still the only one around.
+				// We don't need to do anything.
+				return nil
+			}
+
+			addr, err := w.bindAddrFromServerDetails(apiDetails)
+			if err != nil {
+				if errors.Is(err, errors.NotFound) {
+					w.cfg.Logger.Infof(err.Error())
```

Comment:
What happens in this case, we simply wait for a new server message and try again?

Reply:
Yes. This logic is unchanged from prior.